containerfiles/Containerfile.ci

```diff
@@ -7,3 +7,6 @@ RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial
 RUN dnf update -y && dnf install -y git python3-pip python3-netaddr make gcc sudo rsync && yum clean all
 RUN mkdir -p /home/zuul/src/github.com/openstack-k8s-operators
 RUN git clone https://github.com/openstack-k8s-operators/install_yamls /home/zuul/src/github.com/openstack-k8s-operators/install_yamls
+RUN dnf clean all
```

Comment:
Just asking, but we have a `yum clean all`, that in most new RHEL based distros  is a symlink to dnf, why not just change the previous `yum clean all` with this one?

Reply:
Yeah no reason, I missed that `yum clean all` on line 7 and will just update it to dnf thanks :)